control-base/src/main/java/fi/nls/oskari/control/data/GetGeoPointDataHandler.java

```diff
@@ -82,8 +82,12 @@ public void handleAction(final ActionParameters params) throws ActionException {
 
 			final OskariLayer layer = mapLayerService.find(layerId);
 			final String layerType = layer.getType();
-
-			if (OskariLayer.TYPE_WMS.equals(layerType)) {
+			//HSY doesn't want GFI from basemaps.
+			boolean isBaseMap = false;
```

Comment:
How about instead of this you modify https://github.com/oskariorg/oskari-server/blob/master/service-map/src/main/java/fi/nls/oskari/map/layer/formatters/LayerJSONFormatterWMS.java#L127 with 

```
        final JSONObject attrs = layer.getAttributes();
        boolean disableGFI = false;
        if(attrs != null) {
            disableGFI = attrs.optBoolean(KEY_ISQUERYABLE);
        }
        JSONHelper.putValue(layerJson, KEY_ISQUERYABLE, disableGFI && capabilities.optBoolean(KEY_ISQUERYABLE));
```
Then you would need to have { "isQueryable" : false } for the layer attributes instead of { "basemap" : true }. This would disable GFI requests on the frontend side automatically and this can be left as is? Similar logic should be done for arcgislayers. At least this needs be configurable somehow instead of always blocking for basemaps.